lib/neopixel/blockingneopixel.cpp

```diff
@@ -1,189 +1,193 @@
-#include "mbed.h"
 #include "blockingneopixel.h"

```

Comment:
This should be `BlockingNeopixel` or `Neopixel_Blocking` to match naming of the rest of the stuff in fw repo